src/AWS.Deploy.Orchestration/LocalUserSettings/LocalUserSettingsEngine.cs

```diff
@@ -67,12 +67,23 @@ public async Task UpdateLastDeployedStack(string stackName, string projectName,
                     }
                     else
                     {
-                        localUserSettings.LastDeployedStacks = new List<LastDeployedStack>() {
-                            new LastDeployedStack(
-                                awsAccountId,
-                                awsRegion,
-                                projectName,
-                                new List<string>() { stackName })};
+                        var currentStack = new LastDeployedStack(
+                            awsAccountId,
+                            awsRegion,
+                            projectName,
+                            new List<string>()
+                            {
+                                stackName
+                            });
+
+                        if (localUserSettings.LastDeployedStacks == null)
+                        {
+                            localUserSettings.LastDeployedStacks = new List<LastDeployedStack>() { currentStack };
```

Comment:
is it worth changing `LastDeployedStacks` to non-nullable with constructor initialization or with default empty collection?

Reply:
Done

Reply:
you can remove the whole null check now